### ISSUE TITLE
PPA repository does not have a release file

### ISSUE THREAD

**Issue by alaanousir** [shogun-toolbox/shogun#4932]
**OS used:**
Kubuntu 18.04

**Problem description:**
Following the installation instructions in the [INSTALL.md](https://github.com/shogun-toolbox/shogun/blob/develop/doc/readme/INSTALL.md) file, the command

`sudo add-apt-repository ppa:shogun-toolbox/stable
`

gives me the following error: 

```
E: The repository 'http://ppa.launchpad.net/shogun-toolbox/stable/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

From what I understand, there is currently no package for Ubuntu 18.04 release. Is a new package going to be released any time soon? 
I believe it should be mentioned in the INSTALL.md file the Ubuntu version that the PPA is for.
I am a GSoC applicant and I was wondering if it is a good idea to add a pull request to add this piece of information in the md file.

**Comment by vigsterkr** [comment]
unfortunately as the error says ppa for 18.04 is not available :(

**Comment by alaanousir** [comment]
That is what I thought. Wouldn't it be a good idea to add the verison in the installation instructions to be more specific? Also, I was wondering if a new package is going to be released for 18.04 any time soon.

**Comment by vigsterkr** [comment]
we haven't released for a while - working on it.... ppa is not something that is easy to maintain :(
